dev/TabView/TabView.cpp

```diff
@@ -428,3 +435,28 @@ winrt::DependencyObject TabView::ContainerFromIndex(int index)
     }
     return nullptr;
 }
+
+void TabView::OnTabViewKeyDown(const winrt::IInspectable&, const winrt::KeyRoutedEventArgs& args)
+{
+    winrt::CoreVirtualKeyStates ctrlState = winrt::CoreWindow::GetForCurrentThread().GetKeyState(winrt::VirtualKey::Control);
+
+    if (IsEnabled()
+        && SelectedItem()
+        && (ctrlState & winrt::CoreVirtualKeyStates::Down) == winrt::CoreVirtualKeyStates::Down)
+    {
+        winrt::VirtualKey key = args.Key();
+
+        if (key == winrt::VirtualKey::F4)
```

Comment:
Consider using UIElement.KeyboardAccelerators to implement this - that'll improve what accelerators get read out with accessibility so Ctrl+F4 is announced, which seems desirable to me.

Reply:
That sounds better than what I have now, but do you have an example of this working? I tried setting it up, but my invoke method never gets called. It looks like this:

winrt::KeyboardAccelerator keyboardAccelerator;
        keyboardAccelerator.Key(winrt::VirtualKey::F4);
        keyboardAccelerator.Modifiers(winrt::VirtualKeyModifiers::Control);
        keyboardAccelerator.Invoked(winrt::auto_revoke, { this, &TabView::OnCtrlF4Invoked });
        KeyboardAccelerators().Append(keyboardAccelerator);

Reply:
This is a lot cleaner than what I had! Thanks for the suggestion.